install_collectd.sh

```diff
@@ -0,0 +1,949 @@
+            OVERWRITE_COLLECTD_CONFIG="yes"
+            APP_CONFIGURE="yes"
+        else
+            APP_CONFIGURE="no"
+            APP_FINISHED="yes"
+            echo
+            echo "The write_tsdb plugin is required to send metrics from collectd to the Wavefront Proxy"
+            echo "Manual setup is required"
+            echo
+        fi
+    fi
+
+    if [ -n "$OVERWRITE_COLLECTD_CONFIG" ]; then
+        if command_exists wget; then
+            FETCHER="wget --quiet -O /tmp/collectd_conf.tar.gz"
+        elif command_exists curl; then
+            FETCHER="curl -L --silent -o /tmp/collectd_conf.tar.gz"
+        else
+            exit_with_failure "Either 'wget' or 'curl' are needed"
+        fi
+        echo_step "  Configuring collectd"
+        $FETCHER https://github.com/kentwang929/install/files/421074/default_collectd_conf.tar.gz >>${INSTALL_LOG} 2>&1
+        echo_success
+        echo_step "  Extracting Configuration Files"
+        if [ ! -d "/etc/collectd" ]; then
+            mkdir -p /etc/collectd
+        fi
+        tar -xf /tmp/collectd_conf.tar.gz -C /etc/collectd >>${INSTALL_LOG} 2>&1
+            if [ "$?" != 0 ]; then
+                exit_with_failure "Failed to extract configuration files"
+            fi
+        echo_success
+
+        case $OPERATING_SYSTEM in
+        REDHAT)
+            echo_step "  Overwriting collectd.conf"	
+            mv /etc/collectd.conf /etc/collectd.conf.bak
+            mv /etc/collectd/collectd.conf /etc/collectd.conf
+            echo_success
+            ;;
+        esac
+        
+        echo_step "  Modifying Configuration File at $COLLECTD_WAVEFRONT_CONF_FILE"
+        # Update the configuration file
+        sed -ri s,Host\\s+.*$,"Host \"$PROXY\"",g $COLLECTD_WAVEFRONT_CONF_FILE
+        sed -ri s,Port\\s+.*$,"Port \"$PROXY_PORT\"",g $COLLECTD_WAVEFRONT_CONF_FILE
+        echo_success
+        echo_step "  Restarting collectd"
+        service collectd restart >>${INSTALL_LOG} 2>&1
+        echo_success
+    fi
+
+    if [ -z "$APP_CONFIGURE" ]; then
+        echo
+        if ask "Would you like to configure collectd plugins for your installed application(s)? " Y; then
+            APP_CONFIGURE="yes"
+        else
+            echo
+            echo "Keeping the default configuration"
+            echo
+            APP_CONFIGURE="no"
+            APP_FINISHED="yes"
+        fi
+    fi
+
+    if [ "$APP_CONFIGURE" == "yes" ]; then
+        if command_exists wget; then
+            FETCHER="wget --quiet -O /tmp/${APP_CONFIGURE_NAME}.tar.gz"
+        elif command_exists curl; then
+            FETCHER="curl -L --silent -o /tmp/${APP_CONFIGURE_NAME}.tar.gz"
+        else
+            exit_with_failure "Either 'wget' or 'curl' are needed"
+        fi
+        echo_step "  Pulling application configuration file"
+        APP_LOCATION="https://github.com/kentwang929/install/files/464761/WF-PCInstaller.tar.gz"
```

Comment:
Yeah... I disabled that part of the installer for now (it never gets called). I want to focus on getting Telegraf installer fully tested before attempting to re-organize an merge Chung's code.